components/library/libwebm/libwebm.p5m

```diff
@@ -0,0 +1,36 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file usr/bin/$(MACH64)/sample path=usr/demo/libwebm/$(MACH64)/sample
+file path=usr/bin/$(MACH64)/sample_muxer
+file path=usr/bin/$(MACH64)/vttdemux
+file path=usr/bin/$(MACH64)/webm2pes
+
+file usr/bin/sample path=usr/demo/libwebm/sample
+file path=usr/bin/sample_muxer
+file path=usr/bin/vttdemux
+file path=usr/bin/webm2pes
+
+file path=usr/lib/$(MACH64)/libwebm.so
+file path=usr/lib/libwebm.so
```

Comment:
According to http://www.webmproject.org/ "The WebM Project is dedicated to developing a high-quality, open video format for the web that's freely available to everyone" with its alliance ecosystem "including Cisco, Google, Intel, Microsoft, Mozilla and Netflix".

It is part of the same family of projects as libvpx (vp8/vp9 video codec) and libwebp (still image files). From what I gather, WebM is a container format for video, audio and text streams similar to Matroska. This libwebm apparently allows developers to manipulate it :)

Some more here:
* https://www.webmproject.org/about/faq/
* https://en.wikipedia.org/wiki/WebM
* https://www.macxdvd.com/mac-dvd-video-converter-how-to/webm-vs-mp4.htm

Headers is a good catch... I mostly had it "make install" and packaged what it delivered into the proto dir. Not sure even which binaries are tools or samples, nor how developers would use this. IIRC it got pulled as dependency I needed for something else while preparing the grounds for games and multimedia programs to be at all buildable on OI/Hipster.